### PR TITLE
Clean up TRT files

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

Clean up TRT files

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-jobs:
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml:1]: Current this yaml file is not set up to a pipeline. If needed, we should rewrite this file with the same docker file as the other TRT pipelines.

File: tools/ci_build/github/linux/run_dockerbuild.sh [hunk 2]
```diff
@@ -96,19 +96,6 @@ elif [ $BUILD_DEVICE = "gpu" ]; then
         $GET_DOCKER_IMAGE_CMD --repository "onnxruntime-$IMAGE" \
             --docker-build-args="--build-arg BASEIMAGE=nvcr.io/nvidia/cuda:11.1.1-cudnn8-devel-${BUILD_OS} --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} --build-arg INSTALL_DEPS_EXTRA_ARGS=\"${INSTALL_DEPS_EXTRA_ARGS}\" --build-arg USE_CONDA=${USE_CONDA} --network=host" \
             --dockerfile Dockerfile.ubuntu_gpu_training --context .
-elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
```
Comment (jywu-msft) [inline tools/ci_build/github/linux/run_dockerbuild.sh:99]: i think we may not want to delete this.
we also want a path to validate nvidia's published ubuntu based tensorrt container images.

File: tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml (hunk 1 above)
Comment (jywu-msft) [reply tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml:1]: @HectorSVC set this up awhile back. can you provide context? Which pipeline was set hooked up to previously?

File: tools/ci_build/github/linux/run_dockerbuild.sh (hunk 2 above)
Comment (snnn) [reply tools/ci_build/github/linux/run_dockerbuild.sh:99]: Then should we have a daily pipeline for it?

Comment (jywu-msft) [reply tools/ci_build/github/linux/run_dockerbuild.sh:99]: yes, it's a TODO

Comment (snnn) [reply tools/ci_build/github/linux/run_dockerbuild.sh:99]: Do you need both tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt and tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt7_1?

Comment (snnn) [reply tools/ci_build/github/linux/run_dockerbuild.sh:99]: Yesterday Alisson/Liqun/Natalie had a meeting with me. I was asked to help clean up unused pipelines and pipeline yaml files . That's why I'm doing this.

File: tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml (hunk 1 above)
Comment (snnn) [reply tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml:1]: Never mind. I just put one at: https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=169056&view=results

**Pull request closed without merging**